ksql-serde/src/main/java/io/confluent/ksql/serde/avro/KsqlGenericRowAvroSerializer.java

```diff
@@ -46,48 +51,55 @@
   ByteArrayOutputStream output;
   Encoder encoder;
   List<Schema.Field> fields;
+  private final boolean isInternal;
 
-  public KsqlGenericRowAvroSerializer(org.apache.kafka.connect.data.Schema schema) {
-    this.schema = schema;
-  }
+  KafkaAvroSerializer kafkaAvroSerializer;
 
-  @Override
-  public void configure(final Map<String, ?> map, final boolean b) {
-    rowSchema = (String) map.get(AVRO_SERDE_SCHEMA_CONFIG);
-    if (rowSchema == null) {
-      throw new SerializationException("Avro schema is not set for the serializer.");
+  public KsqlGenericRowAvroSerializer(org.apache.kafka.connect.data.Schema schema,
```

Comment:
I'm a bit confused about why this file existed before we added avro support?

Reply:
We did have standard avro support before but not with Schema Registry integration. This is from that implementation that I changed it to use Schema Registry.